contrib/docker-integration/README.md

```diff
@@ -26,15 +48,14 @@ between a TLS configuration with a CA and self-signed.
 
 		$  sudo chmod +x /usr/local/bin/docker-compose
```

Comment:
I think I would gladly get rid of stuff like that, and replace by a link to "Install Docker Compose" (reduces duplication, reduces maintenance).